SPEC.md

```diff
@@ -549,6 +549,15 @@ The AC Name Type is used as the primary key for a number of fields in the schema
 The schema validator will ensure that the keys conform to these constraints.
 
 
+## AC Kind Type
+
+An AC Kind cannot be an empty string and must begin and end with an alphanumeric character.
+An AC Kind value matching defined kinds, will have defined compatibility. Undefined kinds will have undefined compatibility.
```

Comment:
I think I get what you mean but it would be nice to expand a bit on what "undefined compatibility" means